app/controllers/application_controller.rb

```diff
@@ -4,6 +4,9 @@ class ApplicationController < ActionController::Base
   before_filter :configure_permitted_parameters, if: :devise_controller?
 
   protected
+  def after_sign_in_path_for(resource)
```

Comment:
Unused method argument - `resource`. If it's necessary, use `_` or `_resource` as an argument name to indicate that it won't be used. You can also write as `after_sign_in_path_for(*)` if you want the method to accept any arguments but don't care about them.